XamarinCommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -2,7 +2,7 @@
 <Project Sdk="MSBuild.Sdk.Extras/2.0.54">
   <PropertyGroup>
     <TargetFrameworks>netstandard1.0;netstandard2.0;netstandard2.1;Xamarin.iOS10;MonoAndroid90;MonoAndroid10.0;Xamarin.TVOS10;Xamarin.WatchOS10;Xamarin.Mac20;tizen40</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">$(TargetFrameworks);uap10.0.16299</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">$(TargetFrameworks);uap10.0.17763;net472</TargetFrameworks>
```

Comment:
**net472** the reason why the build pipeline failed.

@jfversluis do we want to support GTK/WPF? If yes, then we should come up with the solution regarding to **ValueTask**. (The build failed because ValueTask is not available for net472)

Reply:
Can be added with https://www.nuget.org/packages/System.Threading.Tasks.Extensions/

Reply:
Then we should add this nuget for net472

Reply:
give me a minute, I am about to fix it

Reply:
I do not add GTK (net471) target framework. the same for WPF (net5-windows or netcoreapp3.1)

Reply:
I saw that you changed the UWP version, any reason for that?

Reply:
XF recommends version 17763